index.html

```diff
@@ -0,0 +1,41 @@
+<html>
+    <head>
+        <title>CSS Rules</title>
+        <style>
+            #first {
+                padding: 20px;
+                background: red;
+                border-right: 1px solid orange;
+                margin-right: 20px;
+                display: inline-block;
+            }
+            #second {
+                background-color: blue;
+                padding: 30px 10px;
+                border: 4px solid green;
+                display: inline-block;
+            }
+        
+            #third {
+                height: 50px;
+                width: 50px;
+                background: #00aced;
+            }
+            #fourth {
+                display: none
+            }
+        </style>
+    </head>
+    <div>
+        <p id="first">Div one</p>
+    </div>
+    <div>
+        <p id="second">Div two</p>
+    </div>

```

Comment:
They are not displaying inline because you have applied the styles to the paragraphs inside the divs. So the paragraphs are displaying inline inside of the div, but the div itself is still displaying as block.

So here we can fix this by applying the styles directly to the div